depends/README.md

```diff
@@ -25,7 +25,8 @@ Common `host-platform-triplets` for cross compilation are:
 - `i686-pc-linux-gnu` for Linux 32 bit
 - `x86_64-pc-linux-gnu` for x86 Linux
 - `x86_64-w64-mingw32` for Win64
-- `x86_64-apple-darwin18` for macOS
+- `x86_64-apple-darwin18` for Intel macOS
+- `arm-apple-darwin20` for ARM macOS
```

Comment:
Shouldn't that be `aarch64`?

Reply:
When compiling it on a mac, `arm-apple-darwin` works. It should be aarch64, I agree, but somehow it's not.

Reply:
What about for cross-compiling then?